CHANGELOG.md

```diff
@@ -12,7 +12,8 @@
 
 ### Changes
 
-* [#98](https://github.com/rubocop-hq/rubocop-rails/pull/98): Mark `Rails/ActiveRecordAliases` as `SafeAutoCorrect` false and disable autocorrect by default. ([@prathamesh-sonpatki][])
+* [#98](https://github.com/rubocop-hq/rubocop-rails/pull/98): Mark `Rails/SaveBang` as `SafeAutoCorrect` false and disable autocorrect by default. ([@prathamesh-sonpatki][])
```

Comment:
I think you accidentally changed the existing line.

Reply:
Oops 😄